packages/card-service/src/config/app.ts

```diff
@@ -31,6 +31,7 @@ export const Config = {
   trustProxy: envBool('TRUST_PROXY', false),
   env: envString('NODE_ENV', 'development'),
   cardServicePort: envInt('CARD_SERVICE_PORT', 3007),
+  cardPaymentTimeout: envInt('CARD_PAYMENT_TIMEOUT', 30000),
```

Comment:
One more thing, let's also include the unit of measurement in the env var name as well